release-team/role-handbooks/enhancements/README.md

```diff
@@ -164,34 +164,72 @@ It is important that this process be followed and documentation remain up-to-dat
-  - Evaluate if the enhancement satisfies all of the [requirements](https://github.com/kubernetes/sig-release/blob/master/releases/release_phases.md#enhancements-freeze) for inclusion in the current release.
+  - Evaluate if the enhancement satisfies all the [requirements](https://github.com/kubernetes/sig-release/blob/master/releases/release_phases.md#enhancements-freeze) for inclusion in the current release.
     - Comment on the Issue with a status updating using one of the [templates from below](#enhancement-freeze-templates)
-    - Update `Enhancement Status` field for this Enhancement in the Enhancement Tracking Board (`Tracked` or `At Risk`)
-- Reach out to each [SIG on Slack](https://github.com/kubernetes/community/blob/master/sig-list.md) tagging one or more chairs or technical leads to communicate the Enhancements statuses. Example [communication templates](#sig-outreach-templates) can be found below.
+    - Update `Enhancement Status` field for this Enhancement in the Enhancement Tracking Board (`tracked for enhancement freeze` or `at risk for enhancement freeze`)
+    - KEPs targeting `stable` will need to be marked as `implemented` after code PRs are merged and the feature gates are removed. This will need to be verified after the code freeze.
+    - *Note*: The scalability section in the README is required for only Enhancements targeting `beta` or `stable`. 
+- Reach out to each [SIG on Slack](https://github.com/kubernetes/community/blob/master/sig-list.md) tagging one or more chairs or technical leads to communicate the Enhancements statuses. Example [communication templates](#sig-outreach-templates-optional) can be found below.
 - Start syncing with Communications Team on giving an induction what's coming up for the release.
 - Send an email to [Kubernetes-Dev](https://groups.google.com/a/kubernetes.io/g/dev) that Enhancement freeze is coming and share current Enhancements status. Examples [1](https://groups.google.com/g/kubernetes-dev/c/-nTNtBBHL2Y/m/WfNzb_E1EAAJ).
 - Provide updates during release team meetings
   - Use the `KEPs by Stage` insight from the Enhancement Tracking Board ([example](https://github.com/orgs/kubernetes/projects/98/insights/3))
 
-#### Week of Enhancements Freeze
+#### PRR Reviews 
+
+The KEP template production readiness questionnaire should be filled out by the KEP authors, and reviewed by the SIG leads. 
+Once the leads are satisfied with both the overall KEP (i.e., it is ready to move to `implementable` state) and the PRR answers,
+the authors request a PRR approval. See [submitting a KEP for production readiness approval](#submitting-a-kep-for-production-readiness-approval) for more details.
+
+When should a KEP owner request for a new PRR?
+- When the KEP is ready to move to `implementable` state from `provisional`.
+- If the KEP is graduating and is missing a section in the README that was not required for earlier stages (e.g. the scalability section is only required for `beta` or `stable`).
```

Comment:
Regarding ^, I've added some info here: https://github.com/kubernetes/enhancements/pull/4162#issuecomment-1695678307

Also, I pinged the PRR Reviewers team to verify/add info. Let's wait until end of the week for feedback. If not, we'll merge this PR with current set of changes and make more changes in a new PR. Thanks!

Regarding ^, I've added more info here: https://github.com/kubernetes/enhancements/pull/4162#issuecomment-1695678307.

Also, pinged the PRR Reviewers team. Let's wait till week-end for feedback. If not, we'll merge this PR with current changes and follow up later. Thanks!